specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,235 @@
+}
+
+// An event emitted when existing SBTs are revoked and burned.
+// Arguments
+// * `owner`: "account.near"
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRevokeLog {
+  owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when a recovery process succeeded to reassign SBT.
+// Arguments
+// * `old_owner`: "old_account.near"
+// * `new_owner`: "new_account.near"
+// * `token_ids`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRecoverLog {
+  old_owner: string;
+  new_owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when a existing tokens are renewed.
+// Arguments
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRenewLog {
+  tokens: uint64[];
+  memo?: string;
+}
+```
+
+Whenever a recovery is made in a way that an existing SBT is burned, the `SbtRevokeLog` event must be emitted.
+
+```typescript
+class SbtBurnedAccounts {
+  burn(account_id) {
+    this.burned_accounts[this.account_id][this.predecessor_id] = true;
+  }
+
+  // query
+  is_burned(account_id, sbt_token_contract): bool {
+    return this.burned_accounts[account_id][sbt_token_id];
+  }
+}
+```
+
+### Recommended functions
+
+Although the functions below are not part of the standard (depending on a use case, they may need different parameters), we recommend them as a part of implementation and we also provide them in the reference implementation.
+These functions should emit appropriate events.
+
+```typescript=
+
+interface SBT {
+
+    /// Function for recovery committee, which can be either
+    /// the issuer, DAO, or an operator smart contract, authorized for the
+    /// recovery process.
+    /// Emits `SbtRecoverLog` when the recover process succeeds.
+    /// Returns an error if the recovery process failed.
+    recover(token_id: uint64, old_address: string, new_address: string): error | undefined;
+
+    /// Function to revoke and burn SBT. Must emit `SbtRevokeLog` event.
+    /// Return true if a token_id is an valid, active SBT. Otherwise returns false.
+    revoke(token_id: uint64): bool;
+}
+```
+
+## Reference Implementation
+
+- https://github.com/alpha-fi/i-am-human/tree/master/contracts/soulbound
+
+## Example Flow
+
+## Consequences
+
+### Positive
+
+- Template and set of guidelines for creating SBT tokens.
+- Ability to create SBT aggregators.
+- Ability to use SBT as a primitive to model non KYC identity, badges, certificates etc...
+- SBT can be further used for "lego" protocols, like: Proof of Humanity (dicussed for NDC Governance), undercollateralized lending, role based authentication sytems, innovative economic and social applications...
+- Stanarized recoverability mechanism.
+- SBT are considered as a basic primitive for Decentralized Societies.
+- new way to implement Sybil attack resistance.
+
+### Netural
+
+- API follows the NEP-171 (NFT) standard.
+  NOTE: we can decide to use `nft_` prefix whenever possible.
```

Comment:
As others have pointed out, mixing these standards and making interfaces compatible is well-intentioned but doomed to cause problems. There are so many examples where NFT standards are poorly or improperly implemented, adding another standard with differing functionality but equal naming in there will cause lots of misclassifications between NFTs/SBTs, and then recover methods called on NFT contracts, SBTs attempted to be listed on NFT marketplaces and so on. Probably nothing will happen because of smart contract panics, but devs will have to put defensive measures into place that will impair interoperability efforts. Users will be annoyed and blame devs all over the place. Everyone will be unhappy.

Reply:
@robert-zaremba Existing tooling will inevitably need to have dedicated support for SBTs, and the role of the standards is to make the support hassle-free. Thus, as we already covered before, it is better to avoid any overlap with the NFT ecosystem to make everybody's lives easier.

Reply:
Actually, I still think that some developers may find it desirable to implement NFT query API on the issuer level.  It is possible, so I would keep it mentioned. I will reword the sentence to make it more clear.